consensus/types/src/beacon_block_body.rs

```diff
@@ -9,9 +9,22 @@ use superstruct::superstruct;
 use test_random_derive::TestRandom;
 use tree_hash_derive::TreeHash;
 
-pub type KzgCommitments<T> =
+pub type KzgCommitments<T> = VariableList<KzgCommitment, <T as EthSpec>::MaxBlobsPerBlock>;
```

Comment:
Hopefully this gets merged https://github.com/ethereum/builder-specs/pull/87

It'd be nice not to have to introduce this type. It makes things quite confusing IMO